Oscillator.cpp

```diff
@@ -0,0 +1,70 @@
+#include "Oscillator.h"
+
+Oscillator::Oscillator()
+{
+    vOctInput = 1.f; defaultVoltage = 0.f; waveformControlPosition = 4; pulseWidthPercent = 50.f; modSocketVolts = 1.f; noteInHz = 440.f; audibleThresholdHz = 20000.f;
+}
```

Comment:
use the member initializer list.
Don't initialize member variables inside constructor bodies.
- [ ] revise